src/test/java/uk/gov/companieshouse/web/accounts/controller/smallfull/CreditorsAfterOneYearControllerTests.java

```diff
@@ -72,6 +89,8 @@ private void setup() {
 
     private static final String ERROR_VIEW = "error";
 
+    private static final String TEST_PATH = "total.currentTotal";
```

Comment:
Can we rename this to be more specific, like `MODEL_VARIABLE` or similar if anyone has a suitable preference?